### ISSUE TITLE
Fast Forward occasionally breaks

### ISSUE THREAD

**Issue by RayleeZ817** [higan-emu/ares#1404]
The fast forward functionality sometimes breaks, in a way that it is always unlocked, and you can't limit it again unless you restart ares.

Video Settings Used:
![image](https://github.com/ares-emulator/ares/assets/104574250/928f2ecc-dd25-4a16-a4ec-70d914eaf3a1)

Controller Settings:
![image](https://github.com/ares-emulator/ares/assets/104574250/57462f06-2554-4ec4-b56a-aeeeb59697aa)

Driver Settings:
![image](https://github.com/ares-emulator/ares/assets/104574250/1f4c00e5-e8f6-4bb0-9253-e4b20cf7df24)

OS: Windows 10
GPU: AMD RX6700XT 
Ares Version: 136 Official

Game: Super Mario 64

**Comment by FitzRoyX** [comment]
Yeah this has been a thing for a long time, even in bsnes if I recall. I can't remember if it's exclusive to wasapi however.
